sectionproperties/analysis/solver.py

```diff
@@ -126,7 +126,6 @@ def function_timer(text, function, *args):
     result = function(*args)
 
     if text != "":
-        print("----completed in {0:.6f} seconds---".format(
-            time.time() - start_time))
+        print("----completed in {0:.6f} seconds---".format(time.time() - start_time))
```

Comment:
```suggestion
        print("----completed in {0:.6f} seconds---\n".format(time.time() - start_time))
```

Suggested along with removing `print("")` lines in all `self.time_info` checks? Add new line at end of this statement being printed to terminal to put line between all time outputs.

Reply:
or
```suggestion
        print(f"----completed in {time.time() - start_time:.6f} seconds---\n")
```